product/roundhouse.tests/sqlsplitters/MySqlStatementSplitterSpecs.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using roundhouse.databases.mysql.parser;
+using roundhouse.sqlsplitters;
+
+namespace roundhouse.tests.sqlsplitters
+{
+    public class MySqlStatementSplitterSpecs
+    {
+        public abstract class concern_for_StatementSplitter : TinySpec
+        {
+            public override void Context()
+            {
+                
+            }
+        }
+
+        [Concern(typeof(StatementSplitter))]
+        public class when_splitting_sql_statements : concern_for_StatementSplitter
+        {
+            public override void Because()
+            {
+
+            }
+
+            [Observation]
+            public void should_split_on_delimiters()
+            {
+                string script = "select * from test1;\nselect * from test2;\n";
+
+                Parser parser = new Parser(script);
+                List<ParsedStatement> statements = parser.Parse();
+                WriteStatements(statements);
+
+                TestContext.Out.WriteLine("Statements parsed: " + statements.Count);
+                Assert.IsTrue(statements[0].Value.Length > 10);
+                Assert.AreEqual(statements.Count, 2);
+            }
+
+            [Observation]
+            public void should_retain_comments()
+            {
+                string script = "/*\n *\n * Test Statement\n *\n */\nselect * from test1;\nselect * from test2;\n";
+
+                Parser parser = new Parser(script);
+                List<ParsedStatement> statements = parser.Parse();
+                WriteStatements(statements);
+
+                TestContext.Out.WriteLine("Statements parsed: " + statements.Count);
+                Assert.IsTrue(statements[0].Value.Length > 25);
+                Assert.AreEqual(statements.Count, 2);
```

Comment:
You've got the expected and actual mixed up here (all of them are). It should be
```suggestion
                Assert.AreEqual(2, statements.Count);
```

Reply:
Okay, I will get these straightened out.